docs/user-journeys/users/application-developer/foundational.md

```diff
@@ -9,80 +9,151 @@ track: "USERS › APPLICATION DEVELOPER › FOUNDATIONAL"
 ---
 
 {% capture overview %}
-If you're an application developer looking to get started with Kubernetes, this page and its linked topics introduce you to foundational concepts and tasks. The focus is on a development workflow -- check out other user journeys to learn about managing production clusters, or providing high availability (HA) for critical application deployments.
+If you're a developer looking to run applications on Kubernetes, this page and its linked topics should help you get started with the fundamentals. Though this level primarily describes development workflows, [subsequent levels](/docs/home/?path=users&persona=app-developer&level=intermediate){:target="_blank"} cover more advanced, production setups.
 
-In a production workflow, it's common to assign the tasks and responsibilities of setting up and maintaining a Kubernetes cluster to the distinct roles of cluster administrator and cluster user. These roles might or might not be assigned to different individuals or teams. As an application developer, you don't need to learn all the intricacies of cluster administration. But you do need to learn the basics, and to understand how certain aspects of administering a cluster can affect development, testing, and deployment at the application level.
+##### **A quick note**
+Generally, app developers can focus more on *what* they're deploying to Kubernetes, rather than *how* the underlying infrastructure works. Though it's possible for a single person to manage both, in many organizations, it’s common to assign the latter to a dedicated {% glossary_tooltip text="cluster operator" term_id="cluster-operator" %}. The cluster operator takes on the responsibility of setting up and maintaining Kubernetes clusters.
+
+As a result, the app developer "user journey" is *not* a comprehensive overview of Kubernetes. It does, however, highlight the aspects of cluster admin that are most relevant to development, testing, and deployment at the application level.
 
```

Comment:
In the previous paragraph, you said "cluster operator." Here you say "cluster admin". Suggestion: Use the same term in both places.